src/Extensions/LanguageConfiguration.re

```diff
@@ -1,14 +1,16 @@
 /*
  * LanguageConfiguration.re
  */
+
 open Oni_Core.Utility;
 module Json = Oni_Core.Json;
 
 module AutoClosingPair = {
   type scopes =
     | String
     | Comment
-    | Other(string);
+    | Other(string)
+    | None;
```

Comment:
It's not a good idea to use `None` as a custom constructor as it conflicts with `option`. Better to use `Nothing` or something like that.

Reply:
Good point - I realized this isn't necessary anyway. I'll take it out.